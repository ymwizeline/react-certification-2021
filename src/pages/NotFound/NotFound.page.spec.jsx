import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import NotFound from './NotFound.page';

describe('NotFound component', () => {
  it('renders properly', () => {
    const element = renderer
      .create(
        <BrowserRouter>
          <NotFound />
        </BrowserRouter>
      )
      .toJSON();

    expect(element).toMatchSnapshot();
  });
});
