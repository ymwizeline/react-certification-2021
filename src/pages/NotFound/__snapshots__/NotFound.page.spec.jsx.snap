// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders properly 1`] = `
<section
  className="not-found"
>
  <a
    className="home-link"
    href="/"
    onClick={[Function]}
  >
    home
  </a>
  <img
    alt="page not found"
    src="404.gif"
  />
</section>
`;
